### ISSUE TITLE
Work type selection modal should gracefully handle n work types

### ISSUE THREAD

**Issue by mjgiarlo** [projecthydra-labs/hyrax#274]
Where n = 15-20 types.

Currently the design we're implementing for work type selection (both for single works and for batch works) in advance of creating new works is a modal with styled radio buttons. This works fine for a small number of work types. The design should accommodate up to 20 work types.

**Comment by mjgiarlo** [comment]
Blocked by projecthydra/sufia#2970.

**Comment by atz** [comment]
projecthydra-labs/hyku#580 is closed, but I don't know that it fully unblocked this.

**Comment by danhorst** [comment]
Duplicate of #190
